spring-webmvc/src/main/java/org/springframework/web/servlet/mvc/method/annotation/ExceptionHandlerExceptionResolver.java

```diff
@@ -28,6 +28,8 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import org.apache.commons.lang.exception.ExceptionUtils;
```

Comment:
Production code should not have a dependency on Apache Commons Lang.

Reply:
I was wondering about that and forgot to ask before submitting the PR.
Going to fix this that today.

Reply:
See my comment on the main issue: There seems to be a simpler approach that I intend to use instead. No need to update the PR if we go with that.